blueocean-dashboard/src/main/js/PipelineRoutes.jsx

```diff
@@ -35,7 +35,7 @@ export default (
                     <Route path="tests" component={RunDetailsTests} />
                     <Route path="artifacts" component={RunDetailsArtifacts} />
                 </Route>
-
+               
```

Comment:
:( normally I wouldn't care about whitespace fixes, but this will cause a conflict that will end up being a pain for me to resolve